src/main/java/dev/slimevr/vr/processor/skeleton/BoneInfo.java

```diff
@@ -5,13 +5,19 @@
 
 public class BoneInfo {
 
+	// TODO(thebutlah): I don't think `BoneType` should include trackers, so
```

Comment:
It includes the whole skeleton, which includes trackers. Idk how you want to omit them while also displaying true skeleton structure... If you want to filter them, you can do it on the client side.

Reply:
The tracker info is already exposed in the `devices` and `synthetic_devices` parts of the data feed. `BoneInfo` should *only* be bones.

Reply:
We'll continue in discord, but i disagree.